src/vsTTempSmooth.cpp

```diff
@@ -1,731 +1,1174 @@
 #include "VCL2/instrset.h"
 #include "vsTTempSmooth.h"
 
+// need forceinline ?
+unsigned int INTABS(int x) { return (x < 0) ? -x : x; }
+
 template <bool pfclip, bool fp>
 template <typename T, bool useDiff>
 void TTempSmooth<pfclip, fp>::filterI(PVideoFrame src[15], PVideoFrame pf[15], PVideoFrame& dst, const int fromFrame, const int toFrame, const int plane) noexcept
 {
-    int src_stride[15]{};
```

Comment:
I use default settings in the Visuals Studio. I remember pinterf also wrote it is not good (?) and make some settings/config-file for Visual Studio so it can auto-load required formatting. It is file https://github.com/DTL2020/mvtools/blob/mvtools-pfmod/.editorconfig . So can I edit this file and add to this project so it will re-format text for your indentation requirements ? Or how to do it manually ?
Is it the lines 
indent_style = space
indent_size = 2
that I need to change indent_size = 4 to keep your requirements ? And to undo changes I need something like select all program text and execute some 'apply new indent' command ?

Reply:
Go to Tools->Options->Text Editor->C/C++->Tabs->Tab size=4 Indent size=4, Insert spaces.
After when you open the file press ctrl+k and then press ctrl+d.